resources/views/employees/form.blade.php

```diff
@@ -41,6 +41,7 @@
             <div class="form-control-wrap ">
                 <div class="form-control-select">
                     <select class="form-control" id="company_id" name="company_id">
+                        <option value="default_option" @if(!isset($employee)) selected @endif>Default Option</option>
```

Comment:
you checked the edit part of this option ?

Reply:
Yes working fine